src/components/tabs/tabs.module.scss

```diff
@@ -0,0 +1,63 @@
+@use "../../styles/abstracts/index" as *;
+
+.root {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.list {
+  display: inline-flex;
+  flex-shrink: 0;
+  background-color: inherit;
+}
+
+.trigger {
```

Comment:
I guess here better to use `tab` instead of `trigger`

Reply:
fix